downloads/api/oai/shipcloud_v1_oai3.json

```diff
@@ -1157,6 +1157,148 @@
+      "get": {
+        "description": "Search pickup dropoff locations by address or geographical coordinates.",
+        "responses": {
+          "200": {
+            "description": "A successful response with a list of pickup dropoff locations.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "pickup_dropoff_locations": {
+                      "type": "array",
+                      "items": {
+                        "$ref": "#/components/schemas/pickup_dropoff_location"
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "$ref": "#/components/responses/400"
+          },
+          "401": {
+            "$ref": "#/components/responses/401"
+          },
+          "402": {
+            "$ref": "#/components/responses/402"
+          },
+          "403": {
+            "$ref": "#/components/responses/403"
+          },
+          "404": {
+            "$ref": "#/components/responses/404"
+          },
+          "500": {
+            "$ref": "#/components/responses/500"
+          }
+        },
+        "parameters": [
+          {
+            "schema": {
+              "type": "string",
+              "example": "dhl, hermes"
+            },
+            "in": "query",
+            "required": true,
+            "name": "carriers",
+            "description": "Must be set with the carrier names (see /carriers endpoint). Multiple carriers can be given as a comma separated list."
+          },
+          {
+            "schema": {
+              "type": "string",
+              "maxLength": 255
+            },
+            "in": "query",
+            "name": "street",
+            "description": "The street with house number and addition if applicable of the address."
+          },
+          {
+            "schema": {
+              "type": "string",
+              "maxLength": 255
+            },
+            "in": "query",
+            "name": "zip_code",
+            "description": "The zipcode of the address."
+          },
+          {
+            "schema": {
+              "type": "string",
+              "maxLength": 255
+            },
+            "in": "query",
+            "name": "city",
+            "description": "The city of the address."
+          },
+          {
+            "schema": {
+              "type": "string",
+              "maxLength": 255
+            },
+            "in": "query",
+            "name": "state",
+            "description": "The state, county or province of the address."
+          },
+          {
+            "schema": {
+              "type": "string",
+              "maxLength": 255
+            },
+            "in": "query",
+            "name": "country",
+            "description": "The country of the address."
+          },
```

Comment:
```suggestion
          {
            "schema": {
              "type": "string",
              "example": "DE",
              "maxLength": 255
            },
            "in": "query",
            "name": "country",
            "description": "The country code of the address."
          },
```

Reply:
Also added a JSON example to the 200 response